### ISSUE TITLE
Unable to generate new certificate

### ISSUE THREAD

**Issue by Rush** [Daplie/letsencrypt-express#74]
```
{ type: 'urn:acme:error:malformed',
detail: 'Parse error reading JWS',
status: 400 }
```

**Comment by coolaj86** [comment]
Hmmm.... OS? node.js version?

I've only seen that (or perhaps another error about JWS) back when I was testing `rsa-compat` before merging it into the master branch of le-acme-core (previously letiny-core). It was due to the private key not matching the account.

Originally I had a bug where the private key conversion JWK <--> B64 corrupted the key, but I'm fairly certain none of that code ever made it to npm and it was a solid week ago.

**Comment by Rush** [comment]
Node.js 6.1/CentOS7 via NVM. It was kinda critical for me to fix it and after removing all node-letsencrypt data, a new cert was issued and problem disappeared.

**Comment by coolaj86** [comment]
Sounds like a corrupt key file.

Well, I'm almost done testing the new letsencrypt-express and have fixed several minor bugs (mostly Windows and node v4.x) in the various other libraries (file path issues, Buffer.from, etc). This is about to go from being the repository that I'm always afraid to look at the issues for to something that I'm actually proud of.

So close...

**Comment by Rush** [comment]
Sounds good man. Looking forward to testing it!

**Comment by coolaj86** [comment]
letsencrypt-express v2.0.0 is finally published (based on node-letsencrypt v2.1.0).

I just published and tested right before that I can retrieve new certificates and also that renewals take place when the appropriate time comes. I did find, however, that I had a bug in letsencrypt-cluster where I was calling the wrong method for renewals, so I need to go back and fix that.

I could really use your help implementing https://github.com/Daplie/le-store-sql

I just added tests.
